solr/CHANGES.txt

```diff
@@ -72,6 +72,10 @@ New Features
 
 Improvements
 ----------------------
+* SOLR-15257: Add DocSetQuery in lieu of DocSet.getTopFilter. This issue led to removing
```

Comment:
We typically add changes entires to the end of the section rather than the beginning.

Reply:
Oh... Thanks for pointing this out. I will revise that

Reply:
Revised!